### DISCUSSION TITLE
To start automatically ?

### DISCUSSION THREAD

**Discussion by mart-e** [photobooth-app/photobooth-app#72] [Q&A] [answered]
Hi,

Thanks for the project. Is this supposed to start automatically at boot or do we need to setup it via a service?
Any recommendation on how to do it?
Thanks

**Comment by mgineer85** [comment]
Since changed to pypi packages there is no installer for automatic start any more. 
but it’s needed so I will add a guide how to again in a few days

**Comment by mart-e** [comment]
Thanks for the quick reply. To make it "reboot proof", I created a small systemd file `/etc/systemd/user/photobooth.service`
```
[Unit]
Description=Photobooth
After=network.target

[Service]
ExecStart=/home/MYUSER/.local/bin/photobooth

[Install]
WantedBy=default.target
```
and enabled it with `$ systemctl --user enable photobooth`
(can probably be improved, not a lot of experience in systemd)

**Comment by mgineer85** [comment, marked as answer]
Hello @mart-e,
I added instructions to the documentation about

- install python service and
- create desktop shortcut as well as
- autostart the webfrontend.

See here:
https://mgrl.github.io/photobooth-docs/installation/#install-photobooth-service-on-startup

I tested quickly and seems to work fine on my installation. Would be happy if you have some feedback that it's working on your installation also.

**Comment by mart-e** [comment]
Thanks, you seems to work better as I did not get proper results with `status` on my file.
Closing as solved :+1:
